cinnabar/tests/test_stats.py

```diff
@@ -211,6 +211,7 @@ def test_confidence_interval_edge_case():
 
     # RMSE (default mode)
     bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="RMSE",
-                              include_true_uncertainty=True,
-                              include_pred_uncertainty=True)
-    assert (bss['low'] > bss['mle']) or (bss['mle'] > bss['high']), error_message
+                              include_true_uncertainty=False,
```

Comment:
I think I'm missing something obvious here - I'm not sure I understand why CI was passing with these being set to True. It sounds like this test (which should reproduce the behaviour of the original) issue isn't working as expected?